fury/tests/test_molecular.py

```diff
@@ -0,0 +1,359 @@
+                            [28.461, 107.741,  71.938],
+                            [26.928, 106.097,  71.983],
+                            [33.507, 106.802,  70.804],
+                            [34.635, 107.689,  70.622],
+                            [35.687, 107.018,  69.765],
+                            [36.530, 107.689,  69.174],
+                            [35.631, 105.690,  69.688],
+                            [36.594, 104.921,  68.903],
+                            [36.061, 104.498,  67.534],
+                            [36.601, 103.580,  66.916],
+                            [37.047, 103.645,  69.660],
+                            [35.907, 102.828,  69.957],
+                            [37.751, 104.014,  70.958]])
+    elements = np.array([7, 6, 6, 8, 6, 6, 6, 8, 7, 7, 6, 6, 8, 7, 6, 6, 8, 6,
+                         8, 6])
+    atom_names = np.array(['N', 'CA', 'C', 'O', 'CB', 'CG', 'CD', 'OE1', 'NE2',
+                           'N', 'CA', 'C', 'O', 'N', 'CA', 'C', 'O', 'CB',
+                           'OG1', 'OG2'])
+    model = np.ones(20)
+    chain = np.ones(20)*65
+    residue_seq = np.ones(20)
+    residue_seq[9:13] = 2
+    residue_seq[13:] = 3
+    residue_seq[6] = 4
+    is_hetatm = np.zeros(20, dtype=bool)
+    secondary_structure = np.array([[65, 1, 65, 3]])
+    colors = np.array([[240/255, 0, 128/255], [1, 1, 0]])
+    for i, color in enumerate(colors):
+        if i:
+            helix = []
+            sheet = secondary_structure
+        else:
+            helix = secondary_structure
+            sheet = []
+        molecule = mol.Molecule(elements, atom_coords, atom_names, model,
+                                residue_seq, chain, sheet, helix, is_hetatm)
+        test_actor = mol.ribbon(molecule)
+        scene.set_camera((28, 113, 74), (34, 106, 70), (-0.37, 0.29, -0.88))
+        scene.add(test_actor)
+        scene.reset_camera()
+        scene.reset_clipping_range()
+
+        if interactive:
+            window.show(scene)
+        npt.assert_equal(scene.GetActors().GetNumberOfItems(), 1)
+        arr = window.snapshot(scene)
+        report = window.analyze_snapshot(arr, colors=[color])
+        npt.assert_equal(report.objects, 1)
+        scene.clear()
+
+
+def test_bounding_box(interactive=True):
+    scene = window.Scene()
+    molecule = mol.Molecule()
+    mol.add_atom(molecule, 6, 0, 0, 0)
+    mol.add_atom(molecule, 6, 1, 1, 1)
+    mol.add_bond(molecule, 0, 1, 1)
+
+    molecule_actor = mol.stick(molecule)
+    test_box = mol.bounding_box(molecule)
+    """
+    scene.add(molecule_actor)
+    scene.add(test_box)
+
+    if interactive:
+        window.show(scene)
+
+    npt.assert_equal(scene.GetActors().GetNumberOfItems(), 2)
+
+    table = mol.PeriodicTable()
+    colors = np.array([table.atom_color(1), table.atom_color(6)])
+    arr = window.snapshot(scene)
+    report = window.analyze_snapshot(arr, colors=colors)
+    npt.assert_equal(report.objects, 1)
+    scene.clear()
+    """
```

Comment:
For some reason, the tests pass only when the actor is not added to the scene. This is strange as the tests pass locally for me without commenting out this section.